usage.txt

```diff
@@ -9,66 +9,64 @@ Usage:
     nodist dist                     List all available node versions.
     nodist ds
 
-    nodist add <version>            Download the specified node version.
-    nodist + <version>
+    nodist add <version>            Install the latest published version matching <version>;
+    nodist + <version>              if any set requirement matches this version, this will get referenced in the future.
 
-    nodist rm <version>             Uninstall the specified node version.
-    nodist - <version>
+    nodist rm <version>             Uninstall the latest installed version matching <version>;
+    nodist - <version>              if any set requirement matches this <version>, it will try to reference another installed version.
     
-    nodist <version>                Use the specified node version globally
-    nodist global <version>         (downloads the executable, if necessary).
+    nodist <ver-req>                Set the global version requirement;
+    nodist global <ver-req>         installs the specified node version if it hasn't been installed.
 
-    nodist local <version>          Use <version> in this directory and its subdirectories.
-                                    (will add a ./node-version file)
+    nodist local <ver-req>          Set the local requirement for this directory and its subdirectories;
+                                    adds or updates the ./node-version file.
     
-    nodist use <version>            Use <version> in the current environment only
-    nodist env <version>            (usually the current terminal window).
+    nodist env <ver-req>            Set the requirement for the current environment only
+                                    (usually the current terminal window).
     
-    nodist run <version> -- <args..>    Run <args> with the specified node version
-    nodist r <version> -- <args..>      (downloads the executable, if necessary).
+    nodist run <ver-req> -- <args>  Run <args> with a version matching the provided requirement.
+    nodist r <ver-req> -- <args>    (Will abort if no matching version is installed)
```

Comment:
I would either lowercase "Will" or capitalize the first word in the subsequent parentheticals.  Also missing period here and on line 40.